supporting_scripts/analysis-of-endpoint-connections/src/main/java/de/tum/cit/endpointanalysis/EndpointAnalyzer.java

```diff
@@ -110,6 +112,27 @@ private static void checkForWildcardEndpoints(EndpointInformation endpoint, List
         }
     }
 
+    /**
+     * Excludes endpoints that are specified in the configuration from the provided EndpointClassInformation list.
+     *
+     * This method performs two main tasks:
+     * 1. Removes entire files that are excluded from the analysis based on the file paths specified in the configuration.
+     * 2. Removes individual endpoints that are excluded from the analysis based on the endpoint URIs specified in the configuration.
+     *
+     * @param endpoints The list of EndpointClassInformation objects to be filtered.
+     */
+    private static void excludeExcludedEndpoints(List<EndpointClassInformation> endpoints) {
+        // remove entire files that are excluded from the analysis
```

Comment:
this comment is unnecessary in my opinion if we explain the method above which includes this comment.